logger.js

```diff
@@ -19,11 +19,11 @@
 const assert = require('assert')
 const jsonHelper = require('./json-helper')
 
-const Logger = (conf, scopedProperties) => {
-  assert.equal(typeof(conf.service), 'string',
+const Logger = (service, conf, scopedProperties) => {
```

Comment:
I would prefer something similar to the Ruby one.

```
const Logger = (service, scopedProperties = {}, now = <default_value), output = <default_value>) => {
```

I personally just don't like passing in conf when there aren't THAT many parameters :)

Reply:
Also the naming as I have edited above to be consistent :)

Reply:
Fixed.